doc/mgr/dashboard.rst

```diff
@@ -281,7 +281,12 @@ Finally, provide the credentials to the dashboard::
   $ ceph dashboard set-rgw-api-secret-key <secret_key>
 
 This is all you have to do to get the Object Gateway management functionality
-working. The host and port of the Object Gateway are determined automatically.
+working. 
```

Comment:
This sentence is duplicated at the beginning of the next paragraph, so it could be dropped?

Reply:
Fixed, thanks for pointing this out!